services/squareone/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
We've been deleting these files (which come from the default Helm starter) since when using Argo CD you'll never see them, and now that the chart is only in Phalanx, it's unlikely anyone would install it manually with Helm and thus see this message.